app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragmentDataModel.kt

```diff
@@ -23,10 +26,15 @@ import org.cru.godtools.shortcuts.GodToolsShortcutManager
 import org.keynote.godtools.android.db.Contract.TranslationTable
 import org.keynote.godtools.android.db.GodToolsDao
 
+private const val KEY_TIPS_LANGUAGE = "KEY_TIPS_LANGUAGE"
+private const val KEY_TIPS_TOOL = "KEY_TIPS_TOOL"
+private const val KEY_TIPS_TYPE = "KEY_TIPS_TYPE"
+
 class ToolDetailsFragmentDataModel @ViewModelInject constructor(
     private val dao: GodToolsDao,
     private val downloadManager: GodToolsDownloadManager,
     manifestManager: ManifestManager,
+    @Assisted private val savedStateHandle: SavedStateHandle,
```

Comment:
just a minor nit, I would put this as the last constructor parameter because it is not a service singleton like everything else